docs/index.md

```diff
@@ -1,98 +1,115 @@
 ---
-title: PMD Introduction
+title: Documentation Index
 keywords: java
-tags: [getting_started]
 permalink: index.html
 toc: false
 summary: >
-    Welcome to PMD, an extensible cross-language static code analyzer.
-    It finds common programming flaws like unused variables, empty catch blocks, unnecessary object creation,
-    and so forth. Additionally it includes CPD, the copy-paste-detector. CPD finds duplicated code.
+    Welcome to the documentation index for PMD and CPD! This page exposes
+    the contents of the documentation site thematically, which you can further
```

Comment:
I find it funny that our index now tells users nothing about *WHAT PMD IS*, but tells them how to use this page....

Reply:
You're right, I added an "Overview" section